svn/trunk/SlickEngagement_Plugin.php

```diff
@@ -391,21 +293,65 @@ public function removeSemicolons($value)
         return str_replace(';', ' ', $value);
     }
 
+    public function getPageBootData() {
+      $siteCode = trim($this->getOption('SiteCode'));
+      if ($siteCode) {
+        global $wp;
+
+        $page_url = (isset($_SERVER['HTTPS']) ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+        $remote = self::defaultServerUrl . '/d/page-boot-data?site=' . $siteCode . '&url=' . rawurlencode($page_url) . '&ts=' . strval(time());
```

Comment:
Let's remove the `&ts=` param. This will negatively impact our systems because every request will have to go to origin (`guild-server`.) I think it's acceptable that if the config changes that we just wait for the cache to expire to see the changes reflected.

Reply:
ok removed